fetch_teleop/src/joystick_teleop.cpp

```diff
@@ -380,7 +379,8 @@ class GripperTeleop : public TeleopComponent
     ros::NodeHandle pnh(nh, name);
 
     // Button mapping
-    pnh.param("button_deadman", deadman_, 10);
+    pnh.param("button_deadman", deadman_body_, 9);
```

Comment:
Got it I will restore that. Is it okay to add other deadman buttons such as deadman_body_, deadman_arm_ ? I added those for convenience.